homeassistant/components/youtube/config_flow.py

```diff
@@ -59,54 +47,38 @@ def extra_authorize_data(self) -> dict[str, Any]:
             "prompt": "consent",
         }
 
-    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
-        """Perform reauth upon an API authentication error."""
-        self.reauth_entry = self.hass.config_entries.async_get_entry(
-            self.context["entry_id"]
-        )
-        return await self.async_step_reauth_confirm()
-
-    async def async_step_reauth_confirm(
-        self, user_input: dict[str, Any] | None = None
-    ) -> FlowResult:
-        """Confirm reauth dialog."""
-        if user_input is None:
-            return self.async_show_form(step_id="reauth_confirm")
-        return await self.async_step_user()
-
     async def async_oauth_create_entry(self, data: dict[str, Any]) -> FlowResult:
         """Create an entry for the flow, or update existing entry."""
-
-        service = build(
-            "youtube",
-            "v3",
-            credentials=Credentials(data[CONF_TOKEN][CONF_ACCESS_TOKEN]),
-        )
-        # pylint: disable=no-member
-        own_channel_request: HttpRequest = service.channels().list(
-            part="snippet", mine=True
-        )
-        response = await self.hass.async_add_executor_job(own_channel_request.execute)
-        user_id = response["items"][0]["id"]
+        try:
+            service = build(
+                "youtube",
+                "v3",
+                credentials=Credentials(data[CONF_TOKEN][CONF_ACCESS_TOKEN]),
+            )
+            # pylint: disable=no-member
+            own_channel_request: HttpRequest = service.channels().list(
+                part="snippet", mine=True
+            )
+            response = await self.hass.async_add_executor_job(
+                own_channel_request.execute
+            )
+            own_channel = response["items"][0]
+        except HttpError as ex:
+            error = ex.reason
+            return self.async_abort(
+                reason="access_not_configured",
+                description_placeholders={"message": error},
+            )
+        except Exception as ex:  # pylint: disable=broad-except
+            LOGGER.error("Unknown error occurred: %s", ex.args)
+            return self.async_abort(reason="unknown")
+        self._own_channel = own_channel
```

Comment:
Could also just be `self._title = own_channel["snippet"]["title"]` since we don't need the entire channel response